docs/ocis/backup.md

```diff
@@ -217,3 +217,12 @@ BACKUP RECOMMENDED/OMITABLE. This folder contains custom web assets. Can be spec
 
 When using an external idp/idm/nats or blobstore, its data needs to be backed up separately. Refer to your idp/idm/nats/blobstore documentation for backup details.
 
+## Backup Consistency command
+
+Infinite Scale now allows checking an existing backup for consistency. Use command
+```bash
+ocis backup consistency -p "<path-to-base-folder>"
+```
+
+`path-to-base-folder` needs to be replaced with the path to the storage providers base path. For example: "/var/lib/ocis/storage/users" or "$HOME/storage/users".
```

Comment:
`...the path to the storage providers base path.` This sentence makes the path origin not clear, you should add the relevant envvar for reference so one can have a look how it was defined.

Reply:
There are multiple envvars for it. Could be `BASE_DATA_BASE` or `STORAGE_USERS_OCIS_ROOT` or even `STORAGE_SYSTEM_OCIS_ROOT`. That is why put two examples instead.

Reply:
The docs say

STORAGE_USERS_OCIS_ROOT

The directory where the filesystem storage will store blobs and metadata. If not defined, the root directory derives from $OCIS_BASE_DATA_PATH:/storage/users.

Referencing an envvar is imho much clearer as we have very good docs about...